samza-core/src/main/scala/org/apache/samza/coordinator/JobModelManager.scala

```diff
@@ -279,9 +276,16 @@ object JobModelManager extends Logging {
         case _ => config
       }
     }
+    val rewrittenConfigs = invokeRegexTopicRewriter(rewrittenConfigs)
+
+    val storageConfig = new StorageConfig(rewrittenConfigs)
+    val sideInputs = storageConfig.getStoreNames.asScala
+      .flatMap(storeName => storageConfig.getSideInputs(storeName).asScala)
+      .map(sideInput => StreamUtil.getSystemStreamFromNameOrId(rewrittenConfigs, sideInput))
+      .toList
 
     // Expand regex input, if a regex-rewriter is defined in config
```

Comment:
minor: this comment needs to be placed above with val rewrittenConfigs = invokeRegexTopicRewriter(rewrittenConfigs)